fv/bpf_test.go

```diff
@@ -617,6 +617,160 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					cc.CheckConnectivity()
 				})
 
+				It("should not be able to spoof workloads IP by another workload", func() {
+					if testOpts.protocol != "udp" {
+						return
+					}
+
+					felixes[0].EnsureBinary("pktgen")
```

Comment:
Could just build all the test binaries into the fat docker image now

Reply:
done